### ISSUE TITLE
Angular.dev ANDI accessibility problem. 

### ISSUE THREAD

**Issue by stot3** [angular/angular#52997]
### Describe the problem that you experienced

https://www.ssa.gov/accessibility/andi/andi.js net::ERR_BLOCKED_BY_RESPONSE.NotSameOriginAfterDefaultedToSameOriginByCoep 200 (OK)

Because of this error, ANDI cannot be run. This blocks the Department of Homeland Security accessibility testing tools. This blocks accessibility nonconformance tools being ran and will severely impact people with disabilities as we are unable to manually audit. Is there a way we can fix this? 

### Enter the URL of the topic with the problem

angular.dev

### Describe what you were looking for in the documentation

We want ANDI tools to be allowed to run an Angular.dev

### Describe the actions that led you to experience the problem

Trying to Run ANDI on angular.dev

### Describe what you want to experience that would fix the problem

Unblocking whatever mechanism is blocking ANDI. 

### If this problem caused an exception or error, please paste it here

```true
https://www.ssa.gov/accessibility/andi/andi.js net::ERR_BLOCKED_BY_RESPONSE.NotSameOriginAfterDefaultedToSameOriginByCoep 200 (OK)
```

**Comment by JoostK** [comment]
For reference, this is because of the `Cross-Origin-Embedder-Policy: require-corp` header.

Could you expand on how this script is executed on this page and what its purpose is?
